enhancements/oc/upgrade_kubernetes.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Upgrading oc to latest version of Kubernetes with gomod
+authors:
+  - "@soltysh"
+reviewers:
+  - "@damemi"
+  - "@ingvagabund"
+  - "@tnozicka"
+  - "@sallyom"
+approvers:
+  - "@mfojtik"
+creation-date: 2019-10-08
+last-updated: 2019-10-08
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Upgrading oc to latest version of Kubernetes with gomod
+
+1. Explicitly opt into go modules, even though we are inside a `GOPATH` directory,
+   otherwise the default auto mechanism turns on, which when it notices vendor
```

Comment:
> `default auto mechanism turns on, which when it notices vendor`

Do you have some upstream doc that provides details on this behavior, i have been looking for something like that but haven't found anything :(

Reply:
https://github.com/golang/go/wiki/Modules#recent-changes is the closest, but that is only for the time of doing the bump. We'll eventually want to turn them off explicitly to ensure we use `vendor` dir contents.